### DISCUSSION TITLE
Adjusting Offset in OCP Profile with Stoichiometry and Determining Initial Electrode Concentrations in LFP Batteries

### DISCUSSION THREAD

**Discussion by jitendraec** [pybamm-team/PyBaMM#4646] [Q&A]
In an LFP battery, how can we adjust the offset in the Open Circuit Voltage (OCP) profile when plotting it with respect to stoichiometry? Also, what method can determine the initial concentrations of both the positive and negative electrodes?

**Comment by brosaplanella** [comment]
Hi! We have now moved our discussions to [Discourse](https://pybamm.discourse.group/). Please post your question there.
